python/sglang/srt/server.py

```diff
@@ -90,9 +90,17 @@
 
 
 @app.get("/health")
-async def health() -> Response:
-    """Health check."""
-    return Response(status_code=200)
+async def health(request: Request) -> Response:
+    """
+    Generate 1 token to verify the health of the inference service.
+    """
+    gri = GenerateReqInput(text="s", sampling_params={"max_new_tokens": 1, "temperature": 0.7})
+    try:
+        _ = await tokenizer_manager.generate_request(gri, request).__anext__()
+        return Response(status_code=200)
+    except Exception as e:
+        logger.exception(e)
+        return Response(status_code=500)
```

Comment:
Why use status code 500?

Reply:
Good question, maybe 503 is better?

Reply:
Yes. I think so. https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/503